src/replication.c

```diff
@@ -3772,7 +3772,7 @@ void replicationCron(void) {
          * match the one stored into 'mf_master_offset' state. */
         int manual_failover_in_progress =
             ((server.cluster_enabled &&
-              server.cluster->mf_end) ||
+              isClusterManualFailoverInProgress()) ||
```

Comment:
@madolson We should refactor the existing cluster failover mechanism to use the standalone one. My thinking is we want the clustering system to mediate the failover, but the data nodes is the one that can actually coordinate and complete the process.

Reply:
I am totally ok with this and hope to get to it one of these days :-)